data/outputs/TEPs/evd68-3C-protease.yaml

```diff
@@ -64,12 +64,10 @@ TEP:
   # List of resources associated with this TEP
   resources:
     # Construct / plasmid resource
-    - name: Enterovirus D68 3C protease construct bearing a C-terminal His-tag that crystallised in the space group P21<sub>21</sub>21
+    - name: D68EV3CPROA-c001 
```

Comment:
@LizbeK : Could we have the plasmid names be something comprehensible to humans? Ideally, it would say whether it was a crystallography construct, a biochemical assay construct, or a biophysical assay construct.

We can add another field for the `id` name you want to assign it for inventory tracking purposes, like:
```yaml
id: D68EV3CPROA-c001
```
Would that be OK?

Reply:
Even better, let's rename it to Scarab ID and then we can give the generic name. I will at some stage show you how amazing that identifier is, we can pull all Scarab data from cloning to PDB deposition with protein made from that plasmid when we know that ID.

The concept "comprehensible to humans" is a touch subject to P2 after we got shut down on our Project code query. But then sending in Frank to chat to Matt is a rookie mistake.

Reply:
What do you need me to do? Go back in YAMLs and change name?

Reply:
Would it be possible to have an `id:` or `scarab_id:` field that matches your Scarab ID, and keep `name:` as a human-readable name that describes the purpose/utility of the resource?

Here's what I'm thinking: Suppose a user wants to grab some resources associated with this completed TEP:
![image](https://user-images.githubusercontent.com/3656088/234663700-e50abb39-ddfa-438f-a20f-72f2a65036a2.png)
The user might expect to find resources like:
* SARS-CoV-2 Mpro recombinant expression construct for biophysical assays
* SARS-CoV-2 Mpro recombinant expression construct for biochemical assays
* SARS-CoV-2 Mpro recombinant expression construct for X-ray fragment screens in space group C 1 2 1 (monomer in asymmetric unit)
* SARS-CoV-2 Mpro recombinant expression construct for X-ray fragment screens in space group P 2 1 2 1 (dimer in asymmetric unit)
* Protein expression and purification protocol for SARS-CoV-2 Mpro biophysical construct

These names help the reader easily navigate which plasmid they want.

Reply:
I've kept the original human-readable `name` but moved these new names to `scarab_id`. We can fix later if this is incorrect!